conf/messages.en

```diff
@@ -171,15 +171,15 @@ businessContactDetails.fullName = Full name
 businessContactDetails.telephoneNumber = Telephone number
 businessContactDetails.emailAddress = Email address
 businessContactDetails.checkYourAnswersLabel = Your contact details
-businessContactDetails.error.fullName.required = Enter a full name
-businessContactDetails.error.telephoneNumber.required = Enter a telephone number
-businessContactDetails.error.emailAddress.required = Enter an email address
-businessContactDetails.error.fullName.length = Full name must be 100 characters or less
-businessContactDetails.error.fullName.invalid = Enter a valid name
-businessContactDetails.error.telephoneNumber.length = Telephone number must be 100 characters or less
-businessContactDetails.error.telephoneNumber.invalid = Enter valid telephone number
-businessContactDetails.error.emailAddress.length = Email address must be 100 characters or less
-businessContactDetails.error.emailAddress.invalid = Enter valid email address
+businessContactDetails.error.fullName.required = Enter your full name or business department
+businessContactDetails.error.telephoneNumber.required = Enter your business telephone number
+businessContactDetails.error.emailAddress.required = Enter your business email address
+businessContactDetails.error.fullName.length = Full name or business department must be 105 characters or less
+businessContactDetails.error.fullName.invalid = Full name or business department must only include letters a to z, numbers, hyphens, spaces and apostrophes
+businessContactDetails.error.telephoneNumber.length = Telephone number must be 20 characters or less
+businessContactDetails.error.telephoneNumber.invalid = Enter a telephone number in the right format, like 01642 123 456, 07777777777 or +33 1 23 45 67 88
```

Comment:
Could we check this error message against the gov.uk design pattern?  I think there are specific example numbers we should use